content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -65,17 +83,18 @@ For Single Page Applications (SPAs), performance metrics will only be available
 [5]: https://www.w3.org/TR/paint-timing/
 [6]: https://www.w3.org/TR/navigation-timing/#sec-navigation-timing
 [7]: https://developer.mozilla.org/en-US/docs/Web/API/History
-[8]: https://www.w3.org/TR/paint-timing/#sec-terminology
-[9]: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming/domInteractive
-[10]: https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event
-[11]: https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event
-[12]: https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event
+[8]: /real_user_monitoring/data_collected/view/#how-is-loading-time-calculated
+[9]: https://www.w3.org/TR/paint-timing/#sec-terminology
+[10]: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming/domInteractive
+[11]: https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event
+[12]: https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event
+[13]: https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event
 {{% /tab %}}
 {{% tab "Resource" %}}
 
 All of your website’s resources are collected by default: images, XHRs, [XMLHttpRequest][1], CSS files, JS assets, and font files.
 
-Detailed network timing data regarding the loading of an application’s resources are being collected with the [Performance Resource Timing API][2].
+Detailed network timing data regarding the loading of an application’s resources are being collected with the [Performance Resource Timing API][2]
```

Comment:
```suggestion
Detailed network timing data for the loading of an application’s resources are collected with the [Performance Resource Timing API][2].
```
Or:
The Performance Resource Timing API][2] collects detailed network timing data for the loading of an application's resources.